manifests/tekton/pipelines/base/okd-coreos-all.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: okd-coreos-all
+spec:
+  params:
+    - default: "https://github.com/openshift/os.git"
+      name: repo
+      type: string
+    - default: "release-4.12"
+      name: branch
+      type: string
+    - default: "scos"
+      name: variant
+      type: string
+    - default: "4.12"
+      name: version
+      type: string
+    - default: "registry.ci.openshift.org/origin/4.12:artifacts"
+      name: rpm-artifacts-image
+      type: string
+  tasks:
+    - name: rpm-artifacts-copy
+      params:
+        - name: image
+          value: $(params.rpm-artifacts-image)
+      taskRef:
+        kind: Task
+        name: rpm-artifacts-copy
+      workspaces:
+        - name: ws
+          workspace: shared-workspace
+    - name: cosa-init
+      params:
+        - name: repo
+          value: $(params.repo)
+        - name: branch
+          value: $(params.branch)
+        - name: variant
+          value: $(params.variant)
+        - name: version
+          value: $(params.version)
+      runAfter:
+        - rpm-artifacts-copy
+      taskRef:
+        kind: Task
+        name: cosa-init
+      workspaces:
+        - name: ws
+          workspace: shared-workspace
+    - name: cosa-build
+      runAfter:
+        - cosa-init
+      taskRef:
+        kind: Task
+        name: cosa-build
+      workspaces:
+        - name: ws
+          workspace: shared-workspace
+    - name: cosa-test
+      runAfter:
+        - cosa-build
+      taskRef:
+        kind: Task
+        name: cosa-test
+      workspaces:
+        - name: ws
+          workspace: shared-workspace
+    - name: cosa-buildextend
```

Comment:
Hi @LorbusChris 
Just for me to understand... I thought this buildextend was part of the scos , and needed to be done before the `cosa-test`

Reply:
the qemu test only needs buildextend-qemu and only does baseline testing - it makes sense to gate other builtextends on passing of that qemu test (and then afterwards run their specific tests also, but we don't do that yet)

Reply:
@LorbusChris - great approach, makes the process easier and allows more flexibility